Tests/MongoSwiftSyncTests/SyncAuthTests.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+@testable import MongoSwift
```

Comment:
I think this is one of those imports that happens to work because `MongoSwift` is a dependency of `MongoSwiftSync`. We should probably update `Package.swift` to have `MongoSwift` as a dependency of `MongoSwiftSyncTests` to explicitly call this out, at least until we publicize the connection string stuff and include it in `MongoSwiftSync`.

As an alternative, could we `@_export` the type in `TestsCommon`? Or does that not work with `internal` types?

Reply:
good point. the` @_exported import` doesn't seem to work for an internal type unfortunately, even if I try doing `@_exported @testable import`.

I added it as an explicit dependency. it was already being imported in a few other files as well.

I also noticed that the one dependency this module had on libmongoc could now be removed because I added the `hosts` property on `ConnectionString`, so I went ahead and did that.